### PR TITLE
Added support for AlmaLinux 8

### PR DESCRIPTION
**Pull request by tparkercbn**
#### Pull Request (PR) description

Added support for AlmaLinux 8

### CODE REVIEW

**Comment by tparkercbn** [comment]
Is anyone able to help me fix the tests?  I'm not very familiar with Travis-CI

**Comment by TheMeier** [comment]
@tparkercbn can you please rebase that to latest master branch

**Comment by tparkercbn** [comment]
> @tparkercbn can you please rebase that to latest master branch

Done I think! :)

Tom

File: metadata.json [hunk 1]
```diff
@@ -70,6 +70,19 @@
       "operatingsystemrelease": [
         "2017"
       ]
+    },
+    {
+      "operatingsystem": "Fedora",
```
Comment (bastelfreak) [inline metadata.json:75]: is that a leftover from the rebase or did you add Fedora on purpose?

**Comment by tparkercbn** [comment]
Leftover.  Let me clean that up.

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by bastelfreak** [comment]
@tparkercbn can you please take a look at the failing tests?

**Comment by tparkercbn** [comment]
> Is anyone able to help me fix the tests? I'm not very familiar with Travis-CI

@bastelfreak I had asked above if someone can help with them.  I have no idea how Travis-CI works.  I am contributing code that I need to use for my job to get the modules to work but right now I'm swamped and don't have time to learn Travis and I'm hoping that someone else who is more familiar with this can help me fix them up.

Tom

**Comment by tparkercbn** [comment]
I pushed a new commit that may help.  I'm just not sure what the consequences are.  

I set: # rubocop:disable RSpec/RepeatedExample anywhere there was a conflict.

**Comment by tparkercbn** [comment]
Looks like it worked!

**Pull request merged**